src/marvin/engine/language_models/azure_openai.py

```diff
@@ -29,3 +33,16 @@ def _get_openai_settings(self) -> dict:
         if marvin.settings.azure_openai.api_version:
             openai_kwargs["api_version"] = marvin.settings.azure_openai.api_version
         return openai_kwargs
+
+    @validator("model", always=True)
```

Comment:
models provided directly to provider-specific `ChatLLMs` do not have the provider prefix e.g. the entire model should be `gpt-35-turbo`